android-components/components/feature/prompts/src/main/java/mozilla/components/feature/prompts/widget/MonthAndYearPicker.kt

```diff
@@ -155,23 +162,19 @@ internal class MonthAndYearPicker @JvmOverloads constructor(
         private const val SPEED_MONTH_SPINNER = 200L
         private const val SPEED_YEAR_SPINNER = 100L
 
-        @VisibleForTesting
-        internal const val DEFAULT_MAX_YEAR = 9999
-
-        @VisibleForTesting
-        internal const val DEFAULT_MIN_YEAR = 1
-
         internal fun getDefaultMinDate(): Calendar {
             return now().apply {
-                month = Calendar.JANUARY
-                year = DEFAULT_MIN_YEAR
+                month = minMonth()
+                year = minYear()
+                day = minDay()
```

Comment:
I think these changes are a bit out of scope for the patch, since we are switching `TimePickerDialogFragment` to not use these functions. That said, these changes look more correct to me. However, I am wondering if perhaps these were intended as a fallback default for when `maximumDate?.toCalendar()` in `TimePickerDialogFragment` was null. Perhaps @Amejia481 could weigh in? It's been a long time but it looks like he was the original author

Reply:
That latest change looks good to me!!